File: Group Code challenge/Question 2/identical.rb
def has_two_identical_letters(string)
    (0..string.length - 2).each do |i|
      return true if string[i] == string[i + 1]
    end
    false
  end
  puts has_two_identical_letters("hello")
  puts has_two_identical_letters("nono")  
  

File: Group Code challenge/Question 2/identical.py
def has_two_identical_letters(string):
    for i in range(len(string) - 1):
        if string[i] == string[i + 1]:
            return True
    return False


print(has_two_identical_letters("hello"))  
print(has_two_identical_letters("nono")) 
print(has_two_identical_letters("sema"))  
print(has_two_identical_letters("trello")) 

